views/js/test/scoring/provider/test.js

```diff
@@ -187,7 +225,7 @@ define([
                 .on('error', function(err){
                     assert.ok(false, 'Got an error : ' + err);
                 })
-                .on('outcome', function(outcomes){
+                .on('outcome', function(outcomes, state){
```

Comment:
state is not tested, is it ok?

Reply:
Good catch, I forgot to remove it during development, but finally I've added additional test cases.